tracer/src/Datadog.Trace/Logging/DirectSubmission/Sink/LogsApi.cs

```diff
@@ -0,0 +1,156 @@
+﻿// <copyright file="LogsApi.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+using Datadog.Trace.Agent;
+
+namespace Datadog.Trace.Logging.DirectSubmission.Sink
+{
+    internal class LogsApi : ILogsApi
+    {
+        internal const string LogIntakePath = "v1/input";
```

Comment:
They have just delivered a v2 api: https://docs.datadoghq.com/api/latest/logs/#send-logs
Also the full spec is here: https://github.com/DataDog/architecture/blob/master/rfcs/event-platform-intake-api-v2/rfc.md

Reply:
>They have just delivered a v2 api

Gah, typical 🤦‍♂️ luckily looks pretty similar, mostly improvements around error handling, so shouldn't be too hard to update. Thanks for pointing that out!

Reply:
Updated the API to v2 in https://github.com/DataDog/dd-trace-dotnet/pull/1945/commits/ab7f1008b0ed153b03b751ce3c3140a1a4b447dd